Slim/Player/Protocols/HTTP.pm

```diff
@@ -467,7 +467,7 @@ sub readPersistentChunk {
 	if ($enhanced->{'status'} == IDLE) {
 		my $readLength = $self->_sysread($_[1], $_[2], $_[3]);
 		$enhanced->{'first'} += $readLength;
-	
```

Comment:
Please remove this file from the change set. It only has a single whitespace change.

Reply:
done